docker-compose.yml

```diff
@@ -31,6 +31,12 @@ services:
       DB_NAME: ${DB_NAME}
       DB_USER: ${DB_USER}
       DB_PASSWORD: ${DB_PASSWORD}
+      SMTP_HOST: ${SMTP_HOST}
+      SMTP_PORT: ${SMTP_PORT}
+      SMTP_USE: ${SMTP_USE}
+      SMTP_SSL: ${SMTP_SSL}
+      SMTP_LOGIN: ${SMTP_LOGIN}
+      SMTP_PASS: ${SMTP_PASS}
```

Comment:
Do we need SMTP? 🤔

Reply:
Do you wants to make own mail server?

Reply:
I don't think we need it, do we?

Reply:
Right now nope, in the future why not?

Reply:
If we don't need it now, there's no reason to add it. We should add code when it's needed

Reply:
Okay, this will be removed in the next update.
The next update is expected when we determine the next steps for working on the project.